ApplicationPattern.yaml

```diff
@@ -5542,53 +5492,25 @@ paths:
           example: 'ro-2-0-0-action-p-000'
     get:
-      operationId: getActionProfileRequest
-      summary: 'Returns the configured path of the Request'
+      operationId: getActionProfileConsequentOperationReference
+      summary: 'Returns the reference on the consequent operation'
       tags:
         - ActionProfile
       security:
         - basicAuth: []
       responses:
         '200':
-          description: 'Request path provided'
+          description: 'Reference on the consequent operation provided'
           content:
             application/json:
               schema:
                 type: object
                 required:
-                  - action-profile-1-0:request
+                  - action-profile-1-0:consequent-operation-reference
                 properties:
-                  action-profile-1-0:request:
-                    type: object
-                    required:
-                      - protocol-reference
-                      - address
-                      - port-reference
-                      - operation-reference
-                    properties:
-                      protocol-reference:
-                        type: string
-                      address:
-                        type: object
-                        minProperties: 1
-                        maxProperties: 1
-                        additionalProperties: false
-                        properties:
-                          ipv-4-address-reference:
-                            type: string
-                          domain-name-reference:
-                            type: string
-                      port-reference:
-                        type: string
-                      operation-reference:
-                        type: string
-                    example:
-                      action-profile-1-0:request:
-                        protocol-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol'
-                        address:
-                          ipv-4-address-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address'
-                        port-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port'
-                        operation-reference: '/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-op-s-bs-002/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-capability/operation-name'
+                  consequent-operation-reference:
```

Comment:
In the required , the attribute name is "action-profile-1-0:consequent-operation-reference" , where as in the properties , it is "consequent-operation-reference".
**Proposing to change the attribute name in the properties from "consequent-operation-reference" to  "action-profile-1-0:consequent-operation-reference"**

Reply:
Changed as proposed.